src/app/zap-templates/zcl/data-model/chip/test-manuf-specific-extension-cluster.xml

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2021 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <cluster manufacturerCode="0xFFF1">
+    <domain>CHIP</domain>
+    <name>Test Manufacturer Specific Extension Cluster Test</name>
+    <code>0xFFF1</code>
+    <define>TEST_MANUF_SPECIFIC_EXTENSION_CLUSTER</define>
+    <description>This is a sample manufacturer specific cluster implementation for testing purposes</description>
+    <!-- Base data types -->
+    <attribute side="server" code="0x0000" define="BOOLEAN" type="BOOLEAN" writable="true" default="false" optional="false">boolean</attribute>
```

Comment:
Adding all the various attributes here seems like it will involve a bunch of up-front implementation work to handle the ones that have types that can't live in the attr store, right?

I'd just add a writable boolean for now and we can expand later as needed.

We should lso have a version of this cluster that does not use the (deprecated, for Matter) `manufacturerCode` bit and just uses explicit 32-bit `code` values, but that can be a separate PR.

We need to enable this in all-clusters-app, right?